config/initializers/rails_admin.rb

```diff
@@ -76,7 +80,14 @@ def edit_user_link
   end
   config.model('MangaCharacter') { parent Manga }
   config.model('MangaStaff') { parent Manga }
-  config.model('Chapter') { parent Manga }
+  config.model 'Chapter' do
+    parent Manga
+    field :manga_id
```

Comment:
Oh, I wasn't really sure what field type to specify, but looking through the docs now I think `:belongs_to_association ` might work?